docs/user-guide/user-roadmap-zowe-cli.md

```diff
@@ -38,18 +36,24 @@ Check out **Medium** for more informative blogs on [Zowe CLI](https://medium.com
 
 :::
 
+## Contributing to Zowe CLI
+
+- Review the [**Contributing guidelines**](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md) for a summary of conventions and best practices for development of Zowe CLI.
+
 ## Zowe CLI community resources
 
 The Zowe ecosystem is more than a collection of applications and extensions. An entire community exists to work on enhancements, help answer questions, and discuss plans for the future of the open source project.
 
-- **If you have an issue** that is specific to Zowe CLI, you can [submit an issue](https://github.com/zowe/zowe-cli/issues/new) against the `zowe-cli` repository on GitHub.
+- Join the [**#zowe-cli**](https://openmainframeproject.slack.com/archives/CC8AALGN6) Slack channel to **ask questions about Zowe CLI**, propose new ideas, and interact with the Zowe community.
 
-- Join the **#zowe-cli** [Slack](https://openmainframeproject.slack.com/) channel to **ask questions about Zowe CLI**, propose new ideas, and interact with the Zowe community. 
+- You can join **Zowe CLI squad meetings** to get involved. The meeting schedule is posted in the [Zowe calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar).
```

Comment:
Same comment as above - the new calendar link is https://zoom-lfx.platform.linuxfoundation.org/meetings/zowe